clustering/infinispan/embedded/spi/pom.xml

```diff
@@ -46,8 +46,8 @@
 
     <dependencies>
         <dependency>
-            <groupId>org.jboss.spec.javax.annotation</groupId>
-            <artifactId>jboss-annotations-api_1.3_spec</artifactId>
+            <groupId>jakarta.annotation</groupId>
+            <artifactId>jakarta.annotation-api</artifactId>
```

Comment:
This will break.  Infinispan 13's protostream module requires the javax.annotation package during build time.